### DISCUSSION TITLE
Request for file-handling advice, related to Serenity blog implementation

### DISCUSSION THREAD

**Discussion by LSNicholls** [serenity-is/Serenity#7210] [General]
Hi Serenity people,

Related to this, from https://serenity.is/blog/2016/12/14/hello-world:

>My experience with GitBook so far (for writing documentation) has been mostly positive. I like the idea of using Markdown / Git couple for quick writing and versioning.
>So instead of a database, my choice was to design a simple, file based blog.
 
Even though I am basically a SQL/db person, I do think this blog implementation is pretty cool. :-)  And so was its chicken-and-egg inaugural post.

Here is my question:  _In a file-based implementation, what would be your recommended method of implementation a search feature?_

Thanks in advance for any thoughts.  This is not a critical problem, but I do have a use case for this, although it doesn't happen to be a blog, and I don't know much about handling file-based repositories.

**Comment by volkanceylan** [comment]
We are using Lucene.Net for search in docs

**Reply by LSNicholls** [reply]
Thank you so much, @volkanceylan
